index.html

```diff
@@ -446,17 +446,25 @@
 ********************************************************************************************************************
 
 
-Rainbow-like iridescence effects occur due to interference when a thin refractive film with thickness on the order of the light wavelength is placed on top of a material. To model this, there is assumed to be such a thin-film sitting on top of the base substrate (whether metal or dielectric), parametrized only by its thickness and the film IOR.
-In practice, these effects are most easily incorporated directly into the microfacet BSDFs of both the metal and dielectric-base layers. The currently recommended thin-film model is that of Belcour and Barla [#Belcour2017].
+### Thin-film iridescence
 
-!!! Note: Future work
-   It is currently not physically clear enough how the IOR of the surrounding media affects the thin-film interference, and how the thin-film affects the Fresnel of the adjacent slabs. Moreover, the behavior of the current parametrization is arguably unintuitive, as the effect of the thickness is not obvious. It may be useful to have a more artist-friendly dial of the "iridescence".
+_Iridescence_ is the occurrence of rainbow-like color fringes when a thin dielectric film with thickness on the order of the wavelength of light is placed on top of a material, due to wave interference between the various electromagnetic reflection modes within the film. To model this, there is assumed to be such a thin-film sitting on top of the base substrate (whether metal or dielectric), parametrized only by:
+
+   - the coverage (presence) weight of the film, **`thin_film_weight`**,
+   - the thickness of the film in nanometers, **`thin_film_thickness`**,
+   - the index of refraction (IOR) of the film, **`thin_film_ior`**,
+
+In practice, this wave-optics effect is most easily incorporated directly into the Fresnel factor of the microfacet BSDFs of both the metal and dielectric-base layers. The currently recommended thin-film model is that of Belcour and Barla [#Belcour2017]. The coverage weight functions as a blend between the BSDF with and without the presence of the film, and is necessary since neither the thickness nor the IOR of the film provide a dial of the effect without altering the shape and saturation of the color fringes.
```

Comment:
I don't think it's necessary to defend the parametrisation, so I propose to simply say:
```suggestion
In practice, this wave-optics effect is most easily incorporated directly into the Fresnel factor of the microfacet BSDFs of both the metal and dielectric-base layers. The currently recommended thin-film model is that of Belcour and Barla [#Belcour2017]. The coverage weight functions as a blend between the BSDF with and without the presence of the film, and thus allows to dial the effect without altering the shape and saturation of the color fringes.
```

Reply:
Agreed, though note that "allows to" should be modified to "allows one to" to be grammatical. ("Allows" needs an object which is being allowed to do the thing).

Reply:
Thanks, I've updated the suggestion.